File: Java/Stack/stack.java
public class Node<T>{

    private T value;

    public Node(T value){
        this.value = value;
    }

    public T getValue(){
        return this.value
    }
}

public class Stack{
    private Node[] stackArr;
    private int top;

    public Stack(){
        this.stackArr = (Node[])new Object[32];
        this.top = -1;
    }

    public boolean isEmpty(){
        if (this.top==-1){
            return true;
        } else {
            return false;
        }
    }

    public T peek(){
        if (!(this.isEmpty()){
            return this.stackArr[this.top]
        }
    }

    public void push(T item){

    } 
}

File: Python/Stack/stack.py
class Node:
    def __init__(self, val):
        self.val = val

class Stack:
    def __init__(self):
        self.stack = []
    def is_empty(self):
        if not self.stack:
            return True
        return False
    def peek(self):
        if not self.is_empty:
            return self.stack[-1]
    def push(self, item):
        node = Node(item)
        self.stack.append(node)
    def pop(self):
        if not self.is_empty:
            return self.stack.pop()
